OpenXMLSDK.Engine/Word/ReportEngine/Models/Charts/ChartModel.cs

```diff
@@ -37,12 +33,23 @@ public class ChartModel : BaseElement
         /// <summary>
         /// Indicate if we delete axis for categories
         /// </summary>
+        [Obsolete("Please use CategoriesAxisModel.DeleteAxis instead")]
         public bool DeleteAxeCategory { get; set; }
 
+        /// <summary>
+        /// Indicate if the legend must be displayed
+        /// </summary>
+        public bool ShowLegend { get; set; }
+
         /// <summary>
         /// Legend font family
         /// </summary>
-        public string FontFamilyLegend { get; set; }
+        public string FontFamilyLegend { get; set; } = "Arial";
```

Comment:
Arial by default is mandatory ?
I prefer to not define default font families as you can inherit value from parent objects.

Reply:
Fixed ;)